packages/typescript/smoke-tests/e2e/tests/chat.spec.ts

```
@@ -1,11 +1,95 @@
-import { test, expect } from '@playwright/test'
+import { test, expect, Page } from '@playwright/test'
```

ESLint

[error] Imports "Page" are only used as type.

(`@typescript-eslint/consistent-type-imports`)

---

[error] Member 'expect' of the import declaration should be sorted alphabetically.

(sort-imports)